test/integration/mockiavelli.int.test.ts

```diff
@@ -151,6 +151,63 @@ describe(`Mockiavelli integration [${TEST_LIBRARY}]`, () => {
         expect(response.status).toEqual(200);
     });
 
+    test('wildcard matching - postfix', async () => {
+        ctx.mockiavelli.mock('/example/*', { status: 200 });
+        jest.spyOn(console, 'error').mockImplementation(() => {});
+
+        await expect(
+            ctx.makeRequest('GET', '/example/')
+        ).resolves.toMatchObject({ status: 200 });
+        await expect(
+            ctx.makeRequest('GET', '/example/foo')
+        ).resolves.toMatchObject({ status: 200 });
+        await expect(
+            ctx.makeRequest('GET', '/example/foo/bar.json')
+        ).resolves.toMatchObject({ status: 200 });
+        await expect(
+            ctx.makeRequest('GET', '/not-matching/')
+        ).resolves.toMatchObject({ status: 404 });
+    });
+
+    test('wildcard matching - prefix', async () => {
+        ctx.mockiavelli.mock('*/example/', { status: 200 });
+        jest.spyOn(console, 'error').mockImplementation(() => {});
+
+        await expect(
```

Comment:
maybe also ``http://external-source/api/example/`` ?

Reply:
Will add.

Reply:
Well, that does not work actually. It is because the hostname and path are matched separately, and only the path part is matched using patterns. To change that a bigger refactor would be needed, which is out of scope of this PR